colcon_bundle/verb/utilities.py

```diff
@@ -70,32 +70,24 @@ def update_shebang(path):
                         str_contents = contents.decode()
                     except UnicodeError:
                         continue
-                    py3_replacement_tuple = py3_shebang_regex.subn(
-                        '#!/usr/bin/env python3', str_contents, count=1)
-                    if py3_replacement_tuple[1] > 0:
-                        logger.info('Found shebang in {file_path}'.format_map(
-                            locals()))
-                        file_handle.seek(0)
-                        file_handle.truncate()
-                        file_handle.write(py3_replacement_tuple[0].encode())
-                        continue
-
-                    py_replacement_tuple = py_shebang_regex.subn(
-                        '#!/usr/bin/env python', str_contents, count=1)
-                    if py_replacement_tuple[1] > 0:
-                        logger.info('Found shebang in {file_path}'.format_map(
-                            locals()))
-                        file_handle.seek(0)
-                        file_handle.truncate()
-                        file_handle.write(py_replacement_tuple[0].encode())
-                    sh_replacement_tuple = sh_shebang_regex.subn(
-                        '#!/usr/bin/env sh', str_contents, count=1)
-                    if sh_replacement_tuple[1] > 0:
-                        logger.info('Found shebang in {file_path}'.format_map(
-                            locals()))
+                    shebang_match = shebang_regex.match(str_contents)
+                    if shebang_match:
+                        shebang_str = shebang_match.group(0)
+                        logger.info('Found shebang in {}'.format(file_path))
+                        shebang_command = cmd_regex.search(shebang_str)
+                        if not shebang_command:
+                            raise ValueError(
```

Comment:
@murphm8 does it make sense to `raise` an error here or `continue`?

Reply:
it didn't throw before, so it seems logical to not introduce that behavior with this change